contracts/src/core/EigenDARelayRegistry.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDARelayRegistry} from "../interfaces/IEigenDARelayRegistry.sol";
+import {OwnableUpgradeable} from "@openzeppelin-upgrades/contracts/access/OwnableUpgradeable.sol";
+
+contract EigenDARelayRegistry is IEigenDARelayRegistry, OwnableUpgradeable {
+
+    mapping(uint32 => string) public relayIdToURL;
+    mapping(address => uint32) public relayAddressToId;
```

Comment:
Curious what's a relay address?

Reply:
onchain address of a relay, the thinking here is that in the future we may want this mapping to permission updates from relays